be1-go/channel/chirp/mod_test.go

```diff
@@ -157,6 +157,125 @@ func Test_SendChirp(t *testing.T) {
 	require.Equal(t, checkData64, msg[0].Data)
 }
 
+func Test_DeleteChirp(t *testing.T) {
+	// Create the hub
+	keypair := generateKeyPair(t)
+
+	fakeHub, err := NewfakeHub(keypair.public, nolog, nil)
+	require.NoError(t, err)
+	generalName := root + laoID + social + posts
+	chirpChannelName := root + laoID + social + sender
```

Comment:
If you don't use root, laoID and social separately, you could merge the three constants into one

Reply:
yes, sure! 👍